docs/content/preview/manage/backup-restore/lightweight-db-clone.md

```diff
@@ -0,0 +1,261 @@
+---
+title: Database clone
+headerTitle: Database clone
+linkTitle: Database clone
+description: Clone your database in YugabyteDB for data recovery, development, and testing.
+badges: tp
+menu:
+  preview:
+    identifier: lightweight-db-clone
+    parent: backup-restore
+    weight: 706
+type: docs
+---
+
+Database cloning in YugabyteDB allows you to quickly create an independent copy of your database for data recovery, development, and testing. The process is efficient because the clone starts by using the same data files as the original database. After the initial creation, the cloned database serves reads and writes independently, creates and maintains its own changes as delta files, separately from the original database.
```

Comment:
What are the objections to the `data recovery` word? What I meant is that it can be used to restore deleted data by creating a clone as of a moment in the past, doing forensic analysis to export the deleted data from the clone and import it back to the original database.

Reply:
Sure we can go with data recovery. See my later comment though about why wouldn't I just use PITR? Maybe a brief mention earlier in use case describing what you wrote here in your comment, this flow wasn't clear to me and is more cumbersome than PITR; so maybe also add small description of when/why use this instead of vanilla PITR